src/server/game/Spells/SpellEffects.cpp

```diff
@@ -5876,3 +5877,20 @@ void Spell::EffectGiveArtifactPowerNoBonus(SpellEffIndex /*effIndex*/)
         if (Item* artifact = unitTarget->ToPlayer()->GetItemByGuid(artifactAura->GetCastItemGUID()))
             artifact->GiveArtifactXp(damage, m_CastItem, 0);
 }
+
+void Spell::EffectGiveHonor(SpellEffIndex /*effIndex*/)
+{
+    if (effectHandleMode != SPELL_EFFECT_HANDLE_HIT_TARGET)
+        return;
+
+    if (!unitTarget || unitTarget->GetTypeId() != TYPEID_PLAYER)
+        return;
+
+    WorldPackets::Combat::PvPCredit packet;
+    packet.Honor = damage;
+    packet.OriginalHonor = damage;
```

Comment:
```
Player::RewardHonor
```

Uses the same value for both

Don't think I have a sniff from retail for it :/
Also they might have changed something for 7.2 as rank isn't showed to the client anymore. When you kill an enemy player, you just get the message that you gained honor (no player rank message given).